src/views/v2/commerce/checkout/cart/billing_identification.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * Tickets Commerce: Billing Identification Fields
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/tickets/v2/commerce/checkout/cart/billing_identification.php
```

Comment:
```suggestion
 * [your-theme]/tribe/tickets/v2/commerce/checkout/cart/billing-identification.php
```
We need to update the file name.